packages/terraform/README.md

```diff
@@ -19,6 +19,29 @@ nx g @nx-extend/terraform:init
 
 ## Usage
 
+### Setup
+```sh
+nx g @nx-extend/terraform:init
+```
+
+#### Available options:
+
+| name              | type     | required | default | description                                                      |
+|-------------------|----------|----------|---------|------------------------------------------------------------------|
+| **`--name`**      | `string` | `true`   |         | Terraform project name                                           |
+| **`--directory`** | `string` | `false`  |         | A directory where the project is placed, based on the sourceRoot |
+| **`--tags`**      | `string` | `false`  | `empty` | Comma separated list for tags                                    |
+
+### Terraform execution
+
+```sh
+nx run consent-service-infra-accounts-dev:initialize
+nx run consent-service-infra-accounts-dev:providers
+nx run consent-service-infra-accounts-dev:plan
+nx run consent-service-infra-accounts-dev:apply
+nx run consent-service-infra-accounts-dev:destroy
```

Comment:
Maybe rename `consent-service-infra-accounts-dev` to `<terraform app name>`?

Reply:
Sure, just changed it!